neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/impl/CompositeIndexDetails.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.neo4j.impl;
+
+import java.util.List;
+
+import org.hibernate.ogm.datastore.neo4j.query.parsing.cypherdsl.impl.CypherDSL;
+
+import org.neo4j.graphdb.Label;
+
+/**
+ * @author The Viet Nguyen &amp;ntviet18@gmail.com&amp;
+ */
+public class CompositeIndexDetails {
```

Comment:
AFAICS, it also create mono-column indexes. I would call it `NeojIndexSpec` to be consistent with `MongoDBIndexSpec`.

(I'll let @DavideD do a proper review as he's more proficient in Neo4j than me)

Reply:
I think it's consistent with the existing classes in Neo4j, I will keep this in mind and change it later for the classes involved.

Reply:
yes, it's aligned with `UniqueConstraintDetails` in Neo4j module

Reply:
I just realized that this is different than unique constraints. Le't call it `Neo4jIndexSpec` as suggested by @gsmet. This way we also highlight that they are two different concepts